### DISCUSSION TITLE
Import beatmaps from my account

### DISCUSSION THREAD

**Discussion by Villfuk02** [ppy/osu#20124] [Ideas] [answered]
I don't have my old osu! installation, so it would be great to be able to import all beatmaps I have ever played based on the information stored in my account. Right now, I have to go through my top plays one by one and click on download. At least its possible to do in the client :)

**Comment by Theighlin** [comment, marked as answer]
Beatmap sync based on other devices installations or account has been asked multiple times, and it's not planned to be added to the game as of now. What you can do is apply the played filter in the beatmap listing (requires active supporter tag)
